docs/ops/backup-and-recovery.md

```
@@ -0,0 +1,135 @@
+### Scenario 1: Container deleted, volume intact
+
+The named volume survives `docker rm`. Just recreate the container:
+
+```bash
+make vm-deploy
+```
+
+### Scenario 2: Corrupt database
+
+1. Stop the container:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)